src/control/cmd/dmg/pool_test.go

```diff
@@ -117,6 +118,14 @@ func TestPoolCommands(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	eUsr, err := user.Current()
+	if err != nil {
+		t.Fatal(err)
+	}
+	eGrp, err := user.LookupGroupId(eUsr.Gid)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Could we mock it with interfaces instead of using the real effective user/group?

Reply:
yes, will do

Reply:
tried (see reverted commit) but I couldn't inject the mock UserExt into the createPool without changing the tests from using runCmdTests. given this is probably going to be an area for redevelopment I have abandoned that approach (the commit can be reused if we want to hoist any of the externals into common later)

Reply:
Fine with me. Thanks for looking into it.